orchagent/response_publisher.h

```diff
@@ -57,8 +61,29 @@ class ResponsePublisher : public ResponsePublisherInterface
     void setBuffered(bool buffered);
 
   private:
+    struct entry
```

Comment:
>entry [](http://example.com/codeflow?start=11&length=5)

Where can I find HLD?

Reply:
This is just internal implementation details. We don't have HLD for this.

The overall design is to put the DB update operation into a different thread for the response publisher.

In the detailed implementation here, we use a FIFO queue to store the DB update events. The main thread will queue up the event into the queue, and the "DB update thread" will read from the queue and process the DB update.
The "entry" struct here is the "DB update event".

Let me know if you need more details on this.